### PR TITLE
[scripts] [combat-trainer] Remove includes, add prefixes to LootProcess class [1 of 3]

### PR DESCRIPTION
**Pull request by MahtraDR**
As per title.

### CODE REVIEW

File: combat-trainer.lic [hunk 1]
```diff
@@ -364,13 +364,7 @@ class SetupProcess
...
 
 class LootProcess
-  include DRC
-  include DRCI
-  include DRCS
-  include DRCH
-  include DRCMM
-
-  def initialize(settings, equipment_manager)
+   def initialize(settings, equipment_manager)
```
Comment (MahtraDR) [inline combat-trainer.lic:367]:
```suggestion
 def initialize(settings, equipment_manager)
```

File: combat-trainer.lic [hunk 2]
```diff
@@ -364,13 +364,7 @@ class SetupProcess
...
 
 class LootProcess
-  include DRC
-  include DRCI
-  include DRCS
-  include DRCH
-  include DRCMM
-
-  def initialize(settings, equipment_manager)
+ def initialize(settings, equipment_manager)
```
Comment (MahtraDR) [inline combat-trainer.lic:367]:
```suggestion
   def initialize(settings, equipment_manager)
```

**Comment by MahtraDR** [comment]
Note I have the final version of this from https://github.com/rpherbig/dr-scripts/pull/5869 running on my 8, no errors, or problems. They all perform exactly as I expect with FA/Skin/Thano priority, and predictable behaviour.

File: combat-trainer.lic [hunk 3]
```diff
@@ -614,7 +608,7 @@ class LootProcess
       # Occassionally, the thread hasn't updated the left_hand/right_hand variables
       # to know what's currently in your hand and will trash what's in your hand!
       # Do a glance to force the variables to be refreshed before we take action.
-      DRC.bput('glance', 'You glance .*')
+      DRC. DRC.bput('glance', 'You glance .*')
```
Comment (MahtraDR) [inline combat-trainer.lic:611]:
```suggestion
      DRC.bput('glance', 'You glance .*')
```

File: combat-trainer.lic [hunk 4]
```diff
@@ -534,53 +528,53 @@ class LootProcess
 
     special = @loot_specials.find { |x| x['name'] == item }
     if special
-      if bput("get #{item}", 'You pick up', 'There isn\'t any more room', 'You get', 'You need a free hand', 'You just can\'t', 'push you over the item limit', 'You stop as you realize the .* is not yours', 'Stow what', 'already in your inventory') == 'already in your inventory'
-        bput("get other #{item}", 'You pick up', 'You get', 'You need a free hand', 'You just can\'t', 'push you over the item limit', 'You stop as you realize the .* is not yours', 'Stow what', 'already in your inventory')
+      if  DRC.bput("get #{item}", 'You pick up', 'There isn\'t any more room', 'You get', 'You need a free hand', 'You just can\'t', 'push you over the item limit', 'You stop as you realize the .* is not yours', 'Stow what', 'already in your inventory') == 'already in your inventory'
```
Comment (asechrest) [inline combat-trainer.lic:531]:
```suggestion
      if DRC.bput("get #{item}", 'You pick up', 'There isn\'t any more room', 'You get', 'You need a free hand', 'You just can\'t', 'push you over the item limit', 'You stop as you realize the .* is not yours', 'Stow what', 'already in your inventory') == 'already in your inventory'
```
```suggestion
      if  DRC.bput("get #{item}", 'You pick up', 'There isn\'t any more room', 'You get', 'You need a free hand', 'You just can\'t', 'push you over the item limit', 'You stop as you realize the .* is not yours', 'Stow what', 'already in your inventory') == 'already in your inventory'
```

**Comment by asechrest** [comment]
@MahtraDR We have leading spaces in seemingly every instance of the DRC prefixes in this PR.

**Comment by MahtraDR** [comment]
> @MahtraDR We have leading spaces in seemingly every instance of the DRC prefixes in this PR.

Check now?

**Comment by asechrest** [comment]
> > @MahtraDR We have leading spaces in seemingly every instance of the DRC prefixes in this PR.
> 
> Check now?

Still have some indentation oddities. Example below, where the bput was just prefixed but then got an indent.

image

**Comment by MahtraDR** [comment]
> > > @MahtraDR We have leading spaces in seemingly every instance of the DRC prefixes in this PR.
> > 
> > 
> > Check now?
> 
> Still have some indentation oddities. Example below, where the bput was just prefixed but then got an indent.
> 
> image

Argh :D

Now it should be ok ;p

**Pull request merged**